framework/src/base/FEProblemBase.C

```diff
@@ -129,6 +129,9 @@ validParams<FEProblemBase>()
                         false,
                         "This causes PETSc to error if it had to reallocate memory in the Jacobian "
                         "matrix due to not having enough nonzeros");
+  params.addParam<bool>("ignore_zeros_in_jacobian",
+                        false,
```

Comment:
What would be the effect on framework, modules and apps if the default was `true`?